.changeset/floppy-tools-accept.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/core': minor
+---
+
+Added catalog discovery mode to ToolSearchProcessor so agents can see every tool id and description while loading schemas on demand.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)